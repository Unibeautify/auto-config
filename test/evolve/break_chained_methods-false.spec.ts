import { testGenetic } from "../testGenetic";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("break_chained_methods=false", {
  beautifiers: [jsBeautify],
  language: "JavaScript",
  originalText: `foo.bar().baz();`,
  desiredText: `foo.bar().baz();`,
});
