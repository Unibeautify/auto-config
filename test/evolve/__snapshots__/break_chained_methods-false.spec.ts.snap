// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`break_chained_methods=false: beautifiedText 1`] = `foo.bar().baz();`;

exports[`break_chained_methods=false: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "JS-Beautify",
    ],
    "brace_style": "collapse",
  },
}
`;

exports[`break_chained_methods=false: fitness 1`] = `0`;
